<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>                </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDesc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLoc">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFindID">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFind">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemoveID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetItems">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelID">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>      ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDesc">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Look in File - Help - About for more details</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuHelp">
     <property name="title">
      <string>Help</string>
     </property>
     <addaction name="actionAbout"/>
    </widget>
    <addaction name="actionExit"/>
    <addaction name="menuHelp"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
